packages/dcos-ui-update-client/src/DCOSUIUpdateClient.ts

```diff
@@ -0,0 +1,57 @@
+import { Observable } from "rxjs";
+import { map } from "rxjs/operators";
+import { request, RequestResponse } from "@dcos/http-service";
+
+import Config from "#SRC/js/config/Config";
+import { getErrorMessage } from "./utils";
+
+import { UIVersionResponse } from "../";
+
+export const DCOSUIUpdateClient = {
+  fetchVersion: (): Observable<RequestResponse<UIVersionResponse>> => {
+    const resp$ = request(
+      `${Config.rootUrl || "/"}dcos-ui-update-service/api/v1/version/`
+    );
+    return resp$.pipe(
+      map((reqResp: RequestResponse<any>) => {
+        if (reqResp.code >= 300) {
+          throw new Error(getErrorMessage(reqResp));
+        }
+        return reqResp;
+      })
+    );
+  },
+  updateUIVersion: (
+    newPackageVersion: string
+  ): Observable<RequestResponse<string>> => {
+    return request(
+      `${Config.rootUrl ||
+        "/"}dcos-ui-update-service/api/v1/update/${newPackageVersion}/`,
+      {
+        method: "POST"
+      }
+    ).pipe(
+      map((reqResp: RequestResponse<any>) => {
+        if (reqResp.code >= 300) {
+          throw new Error(getErrorMessage(reqResp));
+        }
+        return reqResp;
+      })
+    );
+  },
+  resetUIVersion: (): Observable<RequestResponse<string>> => {
+    return request(
+      `${Config.rootUrl || "/"}dcos-ui-update-service/api/v1/reset/`,
+      {
+        method: "DELETE"
+      }
+    ).pipe(
+      map((reqResp: RequestResponse<any>) => {
```

Comment:
Let's stick to `partition` and ditch abbreviations as mentioned in pervious PR 💪